test/03-simple-add-new-form-test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import SimpleAddNewForm from '../src/03/SimpleAddNewForm';
+
+const stub = () => {};
+
+const setup = (customProps = {}) => {
+  const props = Object.assign({}, {
+    title: 'Add new song',
+    inputFieldPlaceholder: 'Enter your favorite song...',
+    handleSubmit: stub
```

Comment:
Nitpick: I've found with the `onFoo`/`handleFoo` pattern having `onSubmit` as the name of the prop is more common.